libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/TimeBasedNextVoiceInstructionsProvider.kt

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.navigation.ui.voice.api
+
+import com.mapbox.api.directions.v5.models.VoiceInstructions
+import com.mapbox.navigation.utils.internal.logW
+
+internal class TimeBasedNextVoiceInstructionsProvider(
+    private val observableTimeSeconds: Int,
```

Comment:
do I understand correctly that it means "from how much seconds ahead voice instructions should be downloaded"?

Reply:
Yes. Your option is more clear but I'm afraid I'm gonna have some MaxLength issues with that name. :)